modules/timeoutRtdProvider.md

```diff
@@ -0,0 +1,137 @@
+ ---
+ layout: page_v2
+ title: Timeout Rtd Module
+ description: Module for managing timeouts in real time
+ page_type: module
+ module_type: rtd
+ module_code : example
+ enable_download : true
+ sidebarType : 1
+ ---
+
+## Overview
+The timeout RTD module enables publishers to dynamically retrieve 
+timeout rules from a timeout provider that determine the timeout based 
+on certain features, or set those rules statically on the page. 
```

Comment:
can you make it clear here that the contents of the response can also be set via config and no endpoint is required?

Reply:
actually you do say this, but perhaps reorder the sentence a small bit

`The timeout RTD module enables publishers to set rules  that determine the timeout based 
on certain features. It supports rule sets dynamically retrieved from a timeout provider as well as rules set directly via configuration`